lib/controllers/ServiceFabrikApiController.js

```diff
@@ -168,16 +168,21 @@ class ServiceFabrikApiController extends BaseController {
   }
 
   getInfo(req, res) {
+    let allDockerImagesRetrieved = false;
     return docker
       .getMissingImages()
-      .then(missingImages => res
-        .status(200)
-        .json({
-          name: this.serviceBrokerName,
-          api_version: this.constructor.version,
-          ready: _.isEmpty(missingImages)
-        })
-      );
+      .then(missingImages => allDockerImagesRetrieved = _.isEmpty(missingImages))
+      .tap(missingImages => logger.info('missing images..', missingImages))
+      .catch(err => logger.info('error occurred while fetching docker images', err))
+      .finally(() => {
+        res.status(CONST.HTTP_STATUS_CODE.OK)
+          .json({
+            name: this.serviceBrokerName,
+            api_version: this.constructor.version,
+            ready: allDockerImagesRetrieved,
+            db_status: this.fabrik.dbManager.getState().status
```

Comment:
can this break where-ever we do not have service-fabrik-mongodb (like non GA landscapes) , since the info endpoint is called from monitoring stack to check if sf is up or not ?

Reply:
No. It returns it as `"db_status":"NOT_CONFIGURED"`

Reply:
Ok , then its fine !